packages/hash/frontend/src/blocks/page/PageBlock.tsx

```diff
@@ -31,6 +32,9 @@ export const PageBlock: VoidFunctionComponent<PageBlockProps> = ({
 }) => {
   const root = useRef<HTMLDivElement>(null);
   const [portals, renderPortal] = usePortals();
+  const [debugging] = useLocalstorageState<
+    { restartCollabButton?: boolean } | boolean
+  >("hash.internal.debugging", false);
```

Comment:
As we’ve started to use prefixed keys, we can also update `const NEW_INSTANCE_KEY = "collab-force-new-instance"` in `packages/hash/frontend/src/blocks/page/collab/EditorConnection.ts`.

Also not sure why we need `{ restartCollabButton?: boolean } | boolean` – that looks like future-proofing we’ll never need. But I don’t think it’s blocking.

Reply:
I thought it might be useful to have a way to turn on *all* debugging options. That's a fair point about the local storage keys, but there's a few that need updating so won't do that here.

Reply:
(Apologies, hit resolve by mistake)